tests/integration/flows/test_sap_rfc_to_adls.py

```diff
@@ -0,0 +1,25 @@
+from viadot.flows import SAPRFCToADLS
+from viadot.config import local_config
+from viadot.sources import AzureDataLake
+
+ADLS_PATH = "raw/supermetrics/mp/test_file_sap.csv"
+
+
+def test_sap_rfc_to_adls():
+    sap_test_creds = local_config.get("SAP").get("QA")
+    flow = SAPRFCToADLS(
+        name="test flow",
+        query_list=[
+            "SELECT MATNR, MATKL FROM MARA WHERE LAEDA LIKE '2022%'",
+            "SELECT MTART, LAEDA FROM MARA WHERE LAEDA LIKE '2022%'",
+        ],
+        func="BBP_RFC_READ_TABLE",
+        sap_credentials=sap_test_creds,
+        local_file_path="test_file.csv",
+        adls_path=ADLS_PATH,
+        overwrite=True,
+    )
+    result = flow.run()
+    assert result.is_successful()
+    file = AzureDataLake(ADLS_PATH)
+    assert file.exists()
```

Comment:
I'm not sure, maybe it would be more clear to specify credentials but some other tests (for example supermetrics to ADLS) don't specify it and I wanted to be consistent